File: Python/A_NivelIniciante/Exercicio_1181.py
def soma(linha,linha2):
 soma = 0
 for x in range(144):
  valor = float(input())
  if(x>=linha and x<linha2):
	  soma+=valor
 return soma
linha = int(input())
linha = linha * 12
linha2 = linha+12 if linha!=0 else 12
somaOuMedia = input()
resultado = soma(linha,linha2) 
print("%.1f" %(resultado if somaOuMedia =="S" else resultado/12))

File: Javascript/A_NivelIniciante/Exercicio_1181.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var linha = 2+(lines[0]*1)*12
var somaOuMedia = lines[1][0]
var soma = 0
for(let i = linha;i<linha+12;i++){
    soma+=parseFloat(lines[i])
}
console.log(somaOuMedia=="S"?soma : (soma/12).toFixed(1))